### ISSUE TITLE
[SF][Button]: [A11y]The Button design type "Emphasized" should be ready out as an additional information, not inside the aria-label

### ISSUE THREAD

**Issue by VidhiNPatel** [UI5/webcomponents#12413]
### Describe the bug

Currently, the button design type, such as "Emphasized," is being read aloud as part of the aria-label. However, according to A11y standards, it should be included in the aria-describedby attribute as additional information.

Moreover, the UI5 sample where the 'Emphasized' detail is provided via the aria-describedby attribute rather than as part of the accessible name. The same approach should be followed here.
https://ui5.sap.com/#/entity/sap.m.Button/sample/sap.m.sample.Button 

### Reproduction steps

1. Go to https://ui5.github.io/webcomponents-react/v2/?path=/docs/inputs-button--docs
2. Change the button design type to 'Emphasized'
3. Observe that aria-label is `Button Text Emphasized`, where 'Emphasized' is unnecessarily included in the name instead of being provided as an additional description.

### Expected Behaviour

According to A11y standards, it should be included in the aria-describedby attribute as additional information.

### UI5 Web Components for React Version

2.14.0

### UI5 Web Components Version

2.14

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by aleksandar-terziev** [comment]
<img width="338" height="176" alt="Image" src="https://github.com/user-attachments/assets/fb025e7a-45dc-4e92-924c-98677aa2aa9d" />

Hello @UI5/ui5-team-ctr-bal , can you take a look at this issue ?

Kind Regards
Aleksandar

**Comment by GongRichard** [comment]
Hi @ilhan007 ,

There still has issue for Emphasized button, please check below screenshot, it will read out as "Default Action"
<img width="1791" height="949" alt="Image" src="https://github.com/user-attachments/assets/9b5e922a-a2de-41bf-b391-c65be12c099e" />

Thanks,
Richard

**Comment by tsanislavgatev** [comment]
Hi @GongRichard ,

Can you please help me understand what the expected behaviour on your end is? What's the expected screen reader output?

Best Regards,
Tsanislav
